src/ekat/ekat_workspace.hpp

```diff
@@ -250,6 +268,10 @@ class WorkspaceManager
   KOKKOS_FORCEINLINE_FUNCTION
   Unmanaged<view_1d<S> > get_space_in_slot(const int team_idx, const int slot) const;
 
+  template <typename S=T>
+  KOKKOS_FORCEINLINE_FUNCTION
+  Unmanaged<view_1d<S> > get_n_spaces_in_slot(const int team_idx, const int slot, const int n) const;
```

Comment:
Might be worth putting a comment above, specifying that the n spaces are contiguous?